promscale/visualize-data/grafana.md

```diff
@@ -0,0 +1,66 @@
+# Connect Promscale and Grafana
+
+To connect Grafana with Promscale, you should have Grafana running. If you do not have Grafana installed, follow the [Grafana installation docs][grafana-install].
+
+This section shows you how to connect Promscale as a Prometheus, Jaeger and PostgreSQL data sources in
+[Grafana][grafana-homepage].
+
+<procedure>
+
+## Promscale as Prometheus datasource
```

Comment:
```suggestion
## Using Promscale as a Prometheus data source
```

Reply:
This is getting resolved when we merge the PR and resolve conflicts  (i.e. this has been fixed in another PR)